app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @post = Post.find(params[:post_id])
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.build(comment_params)
+    @comment.author = current_user
+
+    if @comment.save
+      redirect_to user_post_path(user_id: @post.author.id, id: @post.id), notice: 'Comment added successfully'
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def comment_params
+    params.require(:comment).permit(:text)
```

Comment:
I have looked through the implementation of the form and everything seems good to me but one issue I faced is with testing the forms to ensure there are no errors. When I attempt to run migrations, I get some errors in the console that have to do with renaming a column that doesn't exist. Please take a look at the screenshot below in order to see the error so you can tackle it in order for a reviewer to be able to test the project.

<img width="440" alt="image" src="https://github.com/AleWaweru/blog-App/assets/84930355/f8c437bd-ae10-459e-bf1c-653063fbf96c">